### ISSUE TITLE
Discussions - Post content arrow logic

### ISSUE THREAD

**Issue by gingervik** [daostack/common-backend#247]
1. When users click on a post, the content will be open by default 
<img width="284" alt="Screenshot 2021-03-23 at 14 06 49" src="https://user-images.githubusercontent.com/28015224/112137200-04d67c80-8be1-11eb-9cb5-ddbfd337a095.png">

[zeplin link](https://app.zeplin.io/project/5ef45122508c89515c9d17d8/screen/5ef45498f1a8c7a07d6e71ba)

2. When users click on the arrow and close the post content, it will remain closed when they revisit the same post. If they open it, it will remain open the next time they visit the post.
In other words - it is open by default and then remains the way the user left it, for each post.
<img width="292" alt="Screenshot 2021-03-23 at 14 07 36" src="https://user-images.githubusercontent.com/28015224/112137283-2172b480-8be1-11eb-93cc-55183520ebcd.png">

**Comment by watchtower314** [comment]
demo:
iOS: 

https://user-images.githubusercontent.com/30501647/112464421-dddf8e00-8d39-11eb-9bd7-533e3767c641.mov

android:

https://user-images.githubusercontent.com/30501647/112464464-e9cb5000-8d39-11eb-91c1-9bd8f4408c82.mov

**Comment by kazaryan-t** [comment]
Tested on Staging iOS version 1.9.5 (4978-248)
Tested on Staging Android version 1.9.5 (4978-209)

The post content appears after the user sent a comment to the post.

Steps to reproduce:
1. Navigate to a Discussion and open it
2. Click on the arrow to close the post content
3. Type any message
4. Click on the arrow to send the message

Actual result:
The comment is posted, but in an instant, the post content opens.

Expected result:
The comment is posted, and the post content maintains closed.


https://user-images.githubusercontent.com/68247034/112645600-aebf3e80-8e57-11eb-98ae-542624a58600.MP4


https://user-images.githubusercontent.com/68247034/112645611-b0890200-8e57-11eb-9d17-f2b57237d9ba.mp4

**Comment by watchtower314** [comment]
@lpetkov-sw for some reason, every time we comment, the discussion is being created again from this function
<img width="451" alt="Screen Shot 2021-03-29 at 14 34 36" src="https://user-images.githubusercontent.com/30501647/112883151-e4546980-909b-11eb-8611-76a3e41567c0.png">

I'm a bit lost here because I couldn't find what triggers it, and the fact that we're creating a new discussion, means that we can't get the `isExpanded` value from the discussionStore

**Comment by kazaryan-t** [comment]
Tested on Staging iOS version 1.9.5 (4978-249)
Tested on Staging Android version 1.9.5 (4978-210)

Please, fix the textbox for non-members in the Discussion chats. 

![IMG_0086](https://user-images.githubusercontent.com/68247034/112960647-491bcc80-914d-11eb-8856-5c04615dcd04.PNG)

![IMG_0087](https://user-images.githubusercontent.com/68247034/112960649-4a4cf980-914d-11eb-99dc-fbb714f57045.PNG).

![Screenshot_20210330-110510_common-stg](https://user-images.githubusercontent.com/68247034/112960653-4a4cf980-914d-11eb-896b-54150b8efd30.jpg) 

![Screenshot_20210329-131309_common-stg](https://user-images.githubusercontent.com/68247034/112960655-4ae59000-914d-11eb-9da4-ca169c9b885c.jpg)

**Comment by watchtower314** [comment]
What to test:
- Pretty arrow logic

https://user-images.githubusercontent.com/30501647/113017689-b92a5280-914d-11eb-80b0-72de9cf94efd.mov

- `only members can send message` view fix
<img width="351" alt="Screen Shot 2021-03-30 at 11 27 31" src="https://user-images.githubusercontent.com/30501647/113014809-ec1f1700-914a-11eb-99be-4c42ba8a2984.png">

- space under expanded header
<img width="754" alt="Screen Shot 2021-03-30 at 11 25 11" src="https://user-images.githubusercontent.com/30501647/113014764-df022800-914a-11eb-9864-30d089c8961d.png">

<img width="762" alt="Screen Shot 2021-03-30 at 11 24 58" src="https://user-images.githubusercontent.com/30501647/113014772-e0335500-914a-11eb-9101-171177025b60.png">

- iOS weird behaviour of textInput:
![IMG_7726](https://user-images.githubusercontent.com/30501647/113016775-cd218480-914c-11eb-80b5-3a66d7002ae9.jpg)

- discussion message time alignment
<img width="377" alt="Screen Shot 2021-03-30 at 11 50 59" src="https://user-images.githubusercontent.com/30501647/113018684-c98efd00-914e-11eb-9b7e-17214f1ee376.png">

**Comment by kazaryan-t** [comment]
Tested on Staging iOS version 1.9.5 (4978-255)
Tested on Staging Android version 1.9.5 (4978-216)

1. When the user joins the discussion for the first time, the post content is displayed by default, but if it's long, the user can't scroll it and close clicking on the arrow.

https://user-images.githubusercontent.com/68247034/113145487-236af200-9237-11eb-8f6b-5df0e724d691.MP4

**Comment by watchtower314** [comment]
Demo:

https://user-images.githubusercontent.com/30501647/113186758-0bd73d80-9226-11eb-8fbf-fd7a02dcd633.MOV

**Comment by yanivi71** [comment]
Tested on Staging iOS version 1.9.5 (4978-256)
Tested on Staging Android version 1.9.5 (4978-217)

Found OK
user can scroll and close clicking on the arrow.

**Comment by yanivi71** [comment]
Verified on production
